src/definition/messages/IPostMessageDeleted.ts

```diff
@@ -8,7 +9,7 @@ export interface IPostMessageDeleted {
      * this handler should actually be executed for after the message
      * has been deleted.
      *
-     * @param message The message which was deleted
+     * @param message The deleted message
      * @param read An accessor to the environment
      * @param http An accessor to the outside world
```

Comment:
Any reason you didn't add the context here?

Reply:
I forgot. Adding right now!